docs/content/how-to/aws/create-aws-hosted-cluster-autonode.md

```diff
@@ -0,0 +1,300 @@
+
+To create a hosted cluster with autoNode enabled the Hypershift Operator
+needs to be installed with the [feature gate `--tech-preview-no-upgrade=true`][featuregate].
+
+The Hypershift controller creates and manages the default `openshiftNodeClasses`
+in the hosted cluster allowing you to deploy workloads based in
+your environment with [`NodePools`(`nodepools.karpenter.sh`)][nodepools].
+
+The following steps describes how to install OpenShift workload cluster
+on AWS with AutoNode feature by Karpenter.
+
+## Prerequisites
+
+- [Install the latest and greatest HyperShift CLI](https://hypershift-docs.netlify.app/getting-started/#prerequisites).
+    - Make sure all prerequisites have been satisfied (Pull Secret, Hosted Zone, OIDC Bucket, etc)
+- Ensure that the AWS service-linked role for Spot is enabled in the account where the hosted cluster will be installed. This is a one-time setup per account.
+    - You can verify if the role already exists using the following command:
+```sh
+aws iam get-role --role-name AWSServiceRoleForEC2Spot
+```
+    - If the role does not exist, create it with:
+```sh
+aws iam create-service-linked-role --aws-service-name spot.amazonaws.com
+```
+- Export environment variables, adjusting according to your setup:
+```sh
+# AWS config
+export AWS_CREDS="$HOME/.aws/credentials"
+export AWS_REGION=us-east-1
+
+# OpenShift credentials and configuration
+export CLUSTER_PREFIX=hcp-aws
+export CLUSTER_BASE_DOMAIN=devcluster.openshift.com
+export PULL_SECRET_FILE="${HOME}/.openshift/pull-secret-latest.json"
+export SSH_PUB_KEY_FILE=$HOME/.ssh/id_rsa.pub
+
+## S3 Bucket name hosting the OIDC discovery documents
+# You must have set it up, see Getting Started for more information:
+# https://hypershift-docs.netlify.app/getting-started/
+export OIDC_BUCKET_NAME="${CLUSTER_PREFIX}-oidc"
+```
+
+### Install the Hypershift Operator
+
+This section describes hands on steps to install the Hypershift Operator with AutoNode feature by enabling the feature gate `tech-preview-no-upgrade`. See the following documents for more information:
+
+- [Install HyperShift Operator](https://hypershift-docs.netlify.app/getting-started/#install-hypershift-operator)
+- [Feature Gates][featuregate]
+
+Steps:
+
+- Install the operator:
+```sh
+./hypershift install \
+  --oidc-storage-provider-s3-bucket-name="${OIDC_BUCKET_NAME}" \
+  --oidc-storage-provider-s3-credentials="${AWS_CREDS}" \
+  --oidc-storage-provider-s3-region="${AWS_REGION}" \
+  --tech-preview-no-upgrade=true
+```
+
+- Check if controller is running as expected:
+
+```sh
+oc get all -n hypershift
+```
+
+
+<!-- FIXME: this sectoin is hidden from the official documentation until definiing how to attach a lower privileged policy than Adminsitrator.
+
+### Create Karpenter IAM Role (optional)
+
+Create the IAM Role used by Karpenter controller. This sectoin is extremely optional and must be finetuned to prevent attaching Administrator policy to the role.
+
+!!! warning "Temporary Step"
```

Comment:
there's no need for anything of this, cli will handle any iam on creation for you